src/app/components/authenticated-v2/overlays/armor-tooltip-component/armor-tooltip.component.ts

```diff
@@ -27,6 +28,19 @@ import { ArmorStat, ArmorStatNames } from "../../../../data/enum/armor-stat";
 export class ArmorTooltipComponent {
   @Input() itemTooltip: ResultItem | undefined;
 
+  getSourceText() {
```

Comment:
We should move this to a mapping in the data directory, just in case. You could make a typed dict `EnumDictionary<InventoryArmorSource, string>` and reference it here.